panel/src/components/View/Menu.vue

```diff
@@ -201,6 +211,20 @@ export default {
 	border-end-end-radius: var(--button-rounded);
 }
 
+/* Activation */
+.k-panel-menu .k-activation-button {
+	--button-color-back: var(--color-pink-300);
+	--button-color-text: var(--color-pink-800);
+	border: 1px solid var(--color-pink-400);
+}
+.k-panel-menu .k-activation {
+	display: none;
```

Comment:
On browser sizes smaller than 60rem, the activation bubble is completely hidden. With the hamburger menu hidden, this makes sense, but is there a way to display the bubble when the hamburger menu is active? Or if that won't work layout-wise, maybe fall back to a card?

Reply:
This is by far the most practical approach to it. We can work on a better responsive state later, but considering the time pressure I don't think this is a detail we need to focus on. The mobile version already has the full-width activation button in the menu

Reply:
I agree that we can leave it like this for now. I could see us moving the bubble inside the menu on narrow screens:

<img width="495" alt="Screenshot 2023-11-04 at 17 13 21" src="https://github.com/getkirby/kirby/assets/3788865/25a64816-32bc-40c3-ab18-a99d0ea7cf58">